package/CHANGELOG

```diff
@@ -42,7 +42,6 @@ Changes
 
 Deprecations
 
-
```

Comment:
Non-blocking, but please add this newline back in until the only changes to this doc are  your username above.

Reply:
Done.